### DISCUSSION TITLE
Deactive Short Break Screen

### DISCUSSION THREAD

**Discussion by paivadaniel** [focustimerhq/FocusTimer#683] [Q&A]
Hello, I use the gnome shell extension, and when I click on short break, the screen is stuck on a 10 minute countdown, I logged out Ubuntu and and the count continues, I even restarted my computer and anyway I have to wait for the countdown. How to get out of this count? And how to disable the screen lock?

**Comment by kamilprusko** [comment]
Hi, Do you mean that it gets stuck at 10:00?

A stronger mouse movement or any key will dismiss it. It will get confused when you play a video in background / be on a video meeting it won't work. Hitting Esc key in any case will force it to close.

**Comment by paivadaniel** [comment]
> Hi, Do you mean that it gets stuck at 10:00?
> 
> A stronger mouse movement or any key will dismiss it. It will get confused when you play a video in background / be on a video meeting it won't work. Hitting Esc key in any case will force it to close.

The screen does not stay stuck at 10:00, but counts down until it reaches 00:00. It's like a curse (hehe), if I reset the computer or log out, the 10:00 countdown restarts from zero, I have to wait for it to finish. The solution was to disable the Screen Notifications option. Do you have any idea what it could be? I did tests without background videos, I turned on the computer and the first thing I did was run Pomodoro.

**Reply by kamilprusko** [reply]
In the current version it's supposed to resume the timer after the restart / log-out, and count lost time as pomodoro/break. Unfortunately, there is no setting to control this behaviour.

I'll change this in future releases, likely to pause the timer and ask whether to resume.
